.github/workflows/integration.yml

```diff
@@ -0,0 +1,290 @@
+        with:
+          name: cypress-screenshots
+          path: apps/web/__tests__/report/screenshots
+          if-no-files-found: ignore
+  test-e2e-flakey:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node and Yarn
+        uses: ./.github/actions/setup
+        with:
+          npm-auth-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup environment
+        uses: ./.github/actions/environment
+        with:
+          api-security-token: ${{ secrets.API_SECURITY_TOKEN }}
+          api-endpoint: ${{ vars.API_ENDPOINT_TESTING }}
+          configuration: ${{ vars.CONFIG }}
+
+      - name: Download build
+        uses: actions/download-artifact@v4
+        with:
+          name: build
+
+      - name: Extract build
+        run: tar -xzf build.tar.gz
+
+      - name: E2E tests
+        uses: cypress-io/github-action@v5
+        with:
+          install: false
+          command: yarn test:cypress-quarantine
+
+      - uses: actions/upload-artifact@v4
+        if: failure()
+        with:
+          name: cypress-screenshots
+          path: apps/web/__tests__/report/screenshots
+          if-no-files-found: ignore
+  fitness-web-vitals-mobile:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node and Yarn
+        uses: ./.github/actions/setup
+        with:
+          npm-auth-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup environment
+        uses: ./.github/actions/environment
+        with:
+          api-security-token: ${{ secrets.API_SECURITY_TOKEN }}
+          api-endpoint: ${{ vars.API_ENDPOINT_TESTING }}
+          configuration: ${{ vars.CONFIG }}
+
+      - name: Download build
+        uses: actions/download-artifact@v4
+        with:
+          name: build
+
+      - name: Extract build
+        run: tar -xzf build.tar.gz
+
+      - name: lighthouse mobile audit
+        run: yarn lhci:mobile
+  fitness-web-vitals-desktop:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node and Yarn
+        uses: ./.github/actions/setup
+        with:
+          npm-auth-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup environment
+        uses: ./.github/actions/environment
+        with:
+          api-security-token: ${{ secrets.API_SECURITY_TOKEN }}
+          api-endpoint: ${{ vars.API_ENDPOINT_TESTING }}
+          configuration: ${{ vars.CONFIG }}
+
+      - name: Download build
+        uses: actions/download-artifact@v4
+        with:
+          name: build
+
+      - name: Extract build
+        run: tar -xzf build.tar.gz
+
+      - name: lighthouse desktop audit
+        run: yarn lhci:desktop
+  ready-to-review:
+    runs-on: ubuntu-latest
+    needs: [fitness-compliance, fitness-code-quality, test-unit, build, test-e2e-stable, test-e2e-flakey, fitness-web-vitals-mobile, fitness-web-vitals-desktop]
```

Comment:
it would not require test-e2e-flakey to pass, to notify in chat, right?

Reply:
Now that you mention it yes, thanks for the catch.